docs/source/deepspeed.mdx

```diff
@@ -0,0 +1,104 @@
+<!---
+Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# DeepSpeed for HPUs
+
+[DeepSpeed](https://www.deepspeed.ai/) enables you to fit and train larger models on HPUs thanks to various optimizations.
+In particular, you can use the two following ZeRO configurations that have been validated to be fully functioning with Gaudi:
+- ZeRO-1, which partitions the optimizer states across processes.
+- ZeRO-2, which additionnally partitions the gradients across processes so that each process retains only the gradients corresponding to its portion of the optimizer states.
+
+These configurations also enable you to train your model in *bf16* precision.
```

Comment:
thats not new right?

Reply:
You're right, the wording is not great. I meant that DeepSpeed is compatible with Habana Mixed Precision. I changed it.